pwiz_tools/Skyline/TestFunctional/EditCustomThemeTest.cs

```diff
@@ -155,6 +155,7 @@ protected void TestEditCustomThemes()
                 AssertColor(colors[18], Color.RoyalBlue);
                 OkDialog(editCustomThemeDlg, editCustomThemeDlg.save); // Now save changes
                 Assert.IsNull(ColorScheme.ColorSchemeDemo);
+                // ReSharper disable once HeuristicUnreachableCode

```

Comment:
This is weird. Is this a ReSharper bug?

Reply:
Yeah, it seemed that way to me. Not a great heuristic, clearly